src/tokenOperations/tokenOperationsFactory.ts

```diff
@@ -25,6 +26,8 @@ interface IConfig {
     gasLimitStorePerByte: IGasLimit;
     issueCost: BigNumber.Value;
     esdtContractAddress: IAddress;
+
+    shouldWarnAboutUnsettingBurnRoleGlobally: boolean;
```

Comment:
I wouldn't use a special setting for a single print. Maybe we can have something more general. I don't really expect we would have more warnings, but still it would look better in my opinion not to have it this specific

Reply:
Fixed, now using our existing `Logger` component, with `INFO` level.